content/blog/2021/04/2021-04-07-contributhon-participants.adoc

```diff
@@ -0,0 +1,78 @@
+---
+:layout: post
+:title: "SheCodeAfrica Contributhon participants"
+:tags:
+- events
+- community
+:author: markewaite
+:sig: docs
+opengraph:
+  image: /images/post-images/2020-04-contributhon-participants/shecodeafrica-participants-opengraph.png
+---
+
+The link:https://sites.google.com/shecodeafrica.org/contributhon[SheCodeAfrica Contributhon] started April 1, 2021 with participants from Nigeria, Kenya, and Cameroon.
+Twice weekly mentoring sessions are listed in the link:/events[events calendar].
+We've already received the first pull request to improve the Pipeline documentation of key plugins.
+We're looking forward to more pull requests and more improvements throughout April, 2021.
+
+We'd like to introduce our participants so that you recognize them and can welcome them during code review and online chat.
+
+== Onyinye Ezike
+
+image:/images/post-images/2020-04-contributhon-participants/onyinye-ezike.jpg[Onyinye Ezike, width=400]
```

Comment:
I'd prefer to use floating images as well. I started with floating images alternating left and right but didn't have enough text between the images to prevent them from colliding with each other.  

I didn't try 250 pixel images, but will try them today to see if I can have the floating images that I want with a layout that does not cause pictures to collide with one another at common screen resolutions.